src/frontend/next/src/components/SearchPage/SearchPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { FC, FormEvent, useState } from 'react';
```

Comment:
In [this](https://github.com/vercel/next.js/issues/12964#issuecomment-629642157) thread:

> Next.js automatically adds the React import when JSX is used indeed. However keep in mind that we do still need to import React from 'react' when the React variable is used.